Update cloud release notes for 4-25-24 notes. (#9)
kubecost/docs-cloud: release-notes.md

Kubecost Cloud does not release updates on a scheduled timeline like our self-hosted product. Instead, Kubecost Cloud product updates will be documented here periodically.

## 4/26/24

### New Features:
* Cloud Costs page: performance improvements through pagination and streamlined API calls
* Cloud Costs page: autocomplete now available for filters and aggregations

> * Clusters page: Split Cluster into Cluster Name and Agent ID

Reviewer: ```suggestion
* Clusters page: split Cluster into Cluster Name and Agent ID

```

> * Clusters page: Enhance "Add Cluster" and "Update Cluster" modals

Reviewer: ```suggestion
* Clusters page: enhance "Add Cluster" and "Update Cluster" modals

```

* Improve Azure cost export downloads to be faster and more resilient to failures

### Bug Fixes:
* Fixed issue where costs from the Kubernetes domain could not be added to Collections

> * Fixed issue where the agent version was incorrectly formatted on the Clusters page

Reviewer: ```suggestion
* Fixed issue where the agent version was incorrectly formatted on the Clusters page

```

Kubecost Cloud does not release updates on a scheduled timeline like our self-hosted product. Instead, Kubecost Cloud product updates will be documented here periodically.

> ## 4/25/24

Reviewer: ```suggestion
## 4/26/24
```

### New Features:
* Cloud Costs page: performance improvements through pagination and streamlined API calls
* Cloud Costs page: autocomplete now available for filters and aggregations
* Clusters page: split Cluster into Cluster Name and Agent ID

> 

Reviewer: ```suggestion
```

* Clusters page: enhance "Add Cluster" and "Update Cluster" modals

> 

Reviewer: ```suggestion
```

Kubecost Cloud does not release updates on a scheduled timeline like our self-hosted product. Instead, Kubecost Cloud product updates will be documented here periodically.

## 4/25/24

### New Features:

> * CloudCosts page: performance improvments through pagination and more performant api calls.

Reviewer: ```suggestion
* Cloud Costs page: performance improvements through pagination and more performant API calls
```

> * CloudCosts page: add auto-complete for filters and aggregation by label.

Reviewer: Refactoring this to match with the corresponding release note for Allocations just a few rows down :) 

```suggestion
* Cloud Costs page: autocomplete now available for filters and aggregations
```

> * Clusters page: Split cluster id and cluster name on clusters page.

Reviewer: ```suggestion
* Clusters page: Split Cluster into Cluster Name and Agent ID
```

> * Clusters page: Enhance "Add Cluster" and "Update Cluster" modal.

Reviewer: ```suggestion
* Clusters page: Enhance "Add Cluster" and "Update Cluster" modals
```

> * Enhance azure cloud integrations to have faster and more resilient to failure.

Reviewer: This is an internal improvement, we usually don't mention these, but I'm open to adding it :) 
```suggestion
* Improve Azure cost export downloads to be faster and more resilient to failures

```

### Bug Fixes:

> * Fixed issue with download csv and pdf options in allocations page.

Reviewer: ```suggestion
* Fixed issue where PDF and CSV exports could not be downloaded on the Allocations page
```

Reviewer 2: This isn't actually in yet

> * Fixed issue finding new kubernetes costs to collections.

Reviewer: ```suggestion
* Fixed issue where costs from the Kubernetes domain could not be added to Collections
```

> * Fixed issue with clusters page wrapping agent version.

Reviewer: ```suggestion
* Fixed issue where the agent version was incorrectly formatted on the Clusters page
```

Kubecost Cloud does not release updates on a scheduled timeline like our self-hosted product. Instead, Kubecost Cloud product updates will be documented here periodically.

## 4/25/24

### New Features:
* Cloud Costs page: performance improvements through pagination and streamlined API calls

> 

Reviewer: ```suggestion
```

> * Cloud Costs page: performance improvements through pagination and more performant API calls

Reviewer: I didn't realise we repeated ourselves here with performance/performant
```suggestion
* Cloud Costs page: performance improvements through pagination and streamlined API calls

```

* Cloud Costs page: autocomplete now available for filters and aggregations
* Clusters page: Split Cluster into Cluster Name and Agent ID
* Clusters page: Enhance "Add Cluster" and "Update Cluster" modals
* Improve Azure cost export downloads to be faster and more resilient to failures

> 

Reviewer: ```suggestion

```

### Bug Fixes:

> * Fixed issue where PDF and CSV exports could not be downloaded on the Allocations page

Reviewer: Should remove this as per @aligon ?
```suggestion
```

* Fixed issue where costs from the Kubernetes domain could not be added to Collections
* Fixed issue where the agent version was incorrectly formatted on the Clusters page

> 

Reviewer: ```suggestion

```
